fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1313,13 +1313,13 @@ public class Config extends ConfigBase {
      * kafka util request timeout
      */
     @ConfField(mutable = true)
-    public static long routine_load_kafka_timeout_second = 12;
+    public static long routine_load_kafka_timeout_second = 300;
```

Comment:
I think the `routine_load_kafka_timeout_second` should not be changed, since its value determines the timeout of connection between the BE and Kafka broker.
How about just changing the RPC timeout of `getInfo`?

Reply:
> 

updated